model.py

```diff
@@ -2638,6 +2638,7 @@ def best_open_access_link(self):
         for resource in pool.open_access_links:
             if not any(
                     [resource.representation and
+                     resource.representation.media_type and
                      resource.representation.media_type.startswith(x) 
```

Comment:
Standard Ebooks has some open-access links that have unrecognized media types, which we ignore. This doesn't solve the problem of what to do with those files but it does stop a crash.